pkg/apiserver/apiserver.go

```diff
@@ -280,6 +281,15 @@ func write(statusCode int, apiVersion string, codec runtime.Codec, object runtim
 			return
 		}
 		defer out.Close()
+
+		if wsstream.IsWebSocketRequest(req) {
```

Comment:
Ah, this is to support pod logs, which go a different route through the apiserver than exec does.